src/Aspire.Hosting.AppHost/Aspire.Hosting.AppHost.csproj

```diff
@@ -39,13 +39,20 @@
 
   <ItemGroup>
     <!-- Reference the analyzer to ensure it's built but don't reference its output -->
-    <ProjectReference Include="..\Aspire.Hosting.Analyzers\Aspire.Hosting.Analyzers.csproj" OutputItemType="AnalyzerAssemblyPath" ReferenceOutputAssembly="false" Private="true" />
+    <ProjectReference Include="..\Aspire.Hosting.Analyzers\Aspire.Hosting.Analyzers.csproj" ReferenceOutputAssembly="false" Private="true" />
   </ItemGroup>
 
   <!-- Include the analyzer in the package -->
-  <Target Name="IncludeAnalyzerInPackage" AfterTargets="ResolveProjectReferences">
+  <PropertyGroup>
+    <BeforePack>$(BeforePack);IncludeAnalyzerInPackage</BeforePack>
```

Comment:
Where is this property defined/read? Is this an arcade thing?

Reply:
In arcade: https://github.com/dotnet/arcade/blob/e52018af7f5bab97b0721382d52badfc7edb54f6/src/Microsoft.DotNet.Arcade.Sdk/tools/AfterSigning.proj#L28-L29

Reply:
This is the same approach (roughly) that dotnet/runtime uses to package their analyzers:

https://github.com/dotnet/runtime/blob/3d275542626ac6420269168ecbebe2bcb8e6ae54/eng/packaging.targets#L15

https://github.com/dotnet/runtime/blob/3d275542626ac6420269168ecbebe2bcb8e6ae54/eng/packaging.targets#L136-L152

The only difference is that dotnet/runtime has a Target in the analyzers project `GetAnalyzerPackFiles` to allow for multiple files coming from the analyzer. Here we only care about TargetPath, so I just grab it.